### ISSUE TITLE
Task3.2 - Generate Data for 1 stage addition

### ISSUE THREAD

**Issue by barbaraoramah** [AguaClara/2_stage_coag_addition#5]
In every single circulation, the coag dosage is: 1.1-2.6mg/L, increase 7 times by using the increment function, in which values are derived based on those used by Yingda(Author of our reference thesis). We should add two more data point, one between 1-1.5, and the other between 1.5-2.0. Because usually we generate desirable effluent turbidity around that range, besides, we don't have to start from 0.5mg/L, for that value is too low to treat the raw water.

My plan is that we run 3 circulations per week, basically, that will take 3 days, we use different humic acid between those 3 diff trials within a week. Thus we can test the impact of both humic acid and coagulant dosage on the flocculating process.

**Comment by YuhaoDuuu** [comment]
On Friday, March 2nd, conduct the experiment, added 17g clay and 1g HA into 5L tap water to make the HA&clay mix, and add 6.3ml of lab concentration PACL to make the coagulant solution. The increment function would increase 7 times, let the coag concentration vary from 1.1mg/L to 2.6mg/L. 
Will check the data log on weekend.

**Comment by YuhaoDuuu** [comment]
This week, we kept conducting the 1 stage addition experiment. The effluent turbidity in several trials happened to be around 20NTU, we assumed that it was due to the limit time of our data acquisition state, namely, we did not give the system enough time to form the floc blanket, so we extend the duration of that state, from 3600 seconds to 4800.

**Comment by ChingPangggg** [comment]
1 stage addition experiment was run throughout this week. But we found the the influent turbidity was kept at a level of 2-3 since Wednesday afternoon. On Wednesday afternoon, we changed out the tube between the clay pump and the influent turbidity meter, and we washed out by reversing the direction of the clay pump. However, on Thursday, we found that the influent turbidity was at 2-3 since last night. Therefore, we washed out at a higher speed of the clay pump and the turbidity level went back to approximately 100. No floc or floc blanket was found to be forming in the sedTank. We also inverted the two contact chambers so that they are both in an upflow system now, and the chambers were filled up efficiently.
